src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -377,301 +124,224 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
         }
 
         [Theory]
-        [InlineData("--diff")]
-        [InlineData("-diff")]
-        public void ShouldEnableDiffFeatureWhenPassed(string argName)
+        [InlineData("--verbosity")]
+        [InlineData("-V")]
+        public void ShouldPassLogConsoleArgumentsToStryker_WithLogConsoleArgument(string argName)
         {
-            StrykerOptions actualOptions = null;
-            var runResults = new StrykerRunResult(new StrykerOptions(), 0.3);
+            target.Run(new[] { argName, "Debug" });
 
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>()))
-                .Callback<StrykerOptions, IEnumerable<LogMessage>>((c, m) => actualOptions = c)
-                .Returns(runResults)
-                .Verifiable();
+            strykerRunnerMock.VerifyAll();
 
-            var target = new StrykerCLI(mock.Object);
+            inputs.VerbosityInput.SuppliedInput.ShouldBe(LogEventLevel.Debug.ToString());
+        }
 
+        [Theory]
+        [InlineData("--log-to-file")]
+        [InlineData("-L")]
+        public void ShouldPassLogFileArgumentsToStryker_WithLogLevelFileArgument(string argName)
+        {
             target.Run(new string[] { argName });
 
-            mock.VerifyAll();
+            strykerRunnerMock.VerifyAll();
 
-            actualOptions.DiffEnabled.ShouldBeTrue();
+            inputs.LogToFileInput.SuppliedInput.Value.ShouldBeTrue();
         }
 
         [Theory]
-        [InlineData("--mutation-level")]
-        [InlineData("-level")]
-        public void ShouldSetMutationLevelWhenPassed(string argName)
+        [InlineData("--dev-mode")]
+        public void WithDevModeArgument_ShouldPassDevModeArgumentsToStryker(string argName)
         {
-            var options = new StrykerOptions();
-            var runResults = new StrykerRunResult(options, 0.3);
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>())).Returns(runResults);
+            target.Run(new string[] { argName });
 
-            var target = new StrykerCLI(mock.Object);
+            strykerRunnerMock.VerifyAll();
 
-            target.Run(new string[] { argName, "advanced" });
-            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o =>
-                o.MutationLevel == MutationLevel.Advanced), It.IsAny<IEnumerable<LogMessage>>()));
+            inputs.DevModeInput.SuppliedInput.Value.ShouldBeTrue();
         }
 
         [Theory]
-        [InlineData("--dashboard-compare", "--dashboard-version project")]
-        [InlineData("-compare", "-version project")]
-        public void ShouldEnableDiffCompareToDashboardFeatureWhenPassed(params string[] argName)
+        [InlineData("--concurrency")]
```

Comment:
I think that's not descriptive enough, I would leave `max-concurrent-test-runners` or something like this.

Reply:
I get your point. But I also like the simplicity of `--concurrency`. Maybe we can combine them by using `--max-concurrent-test-runners` and shorthand `-concurrency`? @rouke-broersma what do you think?

Reply:
The concurrency option can later be used for other concurrency features. I don't agree with reverting to max concurrent testrunners.

Reply:
That's on you for sure. Although the two sentences in your last message are somewhat contradicting each other :)

Reply:
Nevermind, pick whatever option :)